### DISCUSSION TITLE
Part category to work on poll

### DISCUSSION THREAD

**Discussion by likeproblem** [likeproblem/RestockRecolour#1] [Polls]
What category you want to me to focus first on?

**Comment by likeproblem** [comment]
RileyHef: more fuel tanks, 25m in particular, and pods.
In the works for parachutes, decouplers and docking port

**Comment by likeproblem** [comment]
Current target: circle pods

**Comment by likeproblem** [comment]
Davidinc: heat shields and shielded docking port

**Comment by likeproblem** [comment]
25m fuel tanks

**Comment by likeproblem** [comment]
step down: mk1, mk1-2, and fuel tanks for those sized

**Comment by likeproblem** [comment]
engine plate

**Comment by likeproblem** [comment]
dart

**Comment by UncleMateo** [comment]
I was thinking of trying to do all the batteries next

**Comment by likeproblem** [comment]
I wanted to start on mainsail but restock screwed me once again, like why do you need another texture and mesh for a single gold foil pipe??

**Reply by UncleMateo** [reply]
I have a vague idea of how to execute the Variant recolors, I almost had them working in my FelotaRecolor for Grounded.  A little more toying around with the config and I think I would have had it.  I think what we should do is make note of parts that are glitchy in the recoloring process, and when we're a bit further along altogether, try to fix each one of those parts, one by one, by applying the (pain in the ass) variant recolors instead.
I wholeheartedly agree that we should try to minimize this for when we truly need to do it, because it's a lot more complicated, as you know.

**Reply by likeproblem** [reply]
The variant method is also useless when you need multiple switches, like for engines mount+engine+shroud etc, since variant method only allows one switcher

**Reply by UncleMateo** [reply]
Well if there's instances where the textures are really broken that might still be a better option.. that's something for deciding when we are closer to being done with most of the parts, I think.  Not a priority just yet.
